lib/src/interfaces/Options.ts

```diff
@@ -1082,6 +1082,13 @@ export interface SideMenuSide {
    * @default true
    */
   shouldStretchDrawer?: boolean;
+  /**
+   * Configure the opening mode of the side menu
+   * #### (iOS specific)
+   * @default 'pushContent'
```

Comment:
Note-to-self: We will likely make `aboveContent` the default in the next major version (8.x.x)